.travis.yml

```diff
@@ -45,7 +45,7 @@ install:
 
 # Create test databases
 before_script:
-  - npm install --global -gulp-cli
+  - npm install --global gulp-cli
```

Comment:
Great job  @reifred.  nit: Moving forward it would be great to add changes that are directly in relation to the commit in that one commit. This change should be included in a commit that best explains what has been done. I don't think has any relation to `[#166185734]Relation between workout log and session`

Reply:
Thanks @owenbob let me rectify this. This happened after rebasing with newly merged PR.